### DISCUSSION TITLE
Roadmap 3 got closed

### DISCUSSION THREAD

**Discussion by MartyIX** [xunit/xunit#3052] [General]
I would leave a simple comment but in #2133 but comments are/were not allowed there. I just wanted to say thank you for the great work! I watch xunit v3 progress from early days and I love your work and looking forward to using xUnit 3. ❤️

**Comment by bradwilson** [comment]
Thanks!

I had closed the commenting on that issue right away because I looked at it like a meta-issue. Time moved forward I'd probably just use the new GitHub Projects feature. 😄
